openassessment/xblock/submission_mixin.py

```diff
@@ -766,7 +766,12 @@ def submission_path_and_context(self):
             context['submit_enabled'] = submit_enabled
             path = "openassessmentblock/response/oa_response.html"
         elif workflow["status"] == "cancelled":
-            context["workflow_cancellation"] = self.get_workflow_cancellation_info(self.submission_uuid)
+            if self.teams_enabled:
+                context["workflow_cancellation"] = self.get_team_workflow_cancellation_info(
+                    workflow["team_submission_uuid"])
```

Comment:
close paren on new line

Reply:
This is more of a stylistic nit:
consider using a ternary operator to get the proper workflow id so we can have fewer lines :)
```
uuid_to_use = workflow["team_submission_uuid"] if self.teams_enabled else self.submission_uuid
context["workflow_cancellation"] = self.get_team_workflow_cancellation_info(uuid_to_use)
```

Reply:
different get_workflow methods as well, though.